platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -781,9 +781,16 @@ data:
   # The hostname and port of the database server.
   server:
 
-  # Name of the database.
+  # The name of the database.
+  # For `sqlite` and `aurora`, this is the path to the database file.
+  # For `postgres`, this is the name of the database.
   database:
 
+  # The schema of the database.
+  # For `postgres` and `aurora`, default is `public`
+  # For `sqlite`, this is ignored.
+  schema:
```

Comment:
What is the expected behavior of setting `schema` when tables have already been created under public?

Reply:
The new set of tables will be created under the specified `schema`. If they set the `schema` to `public`, the existing tables will be used.

Reply:
What happens to in-progress SEP transactions? You can no longer query them since they are in the public schema, right?

Reply:
When leaving as empty, backward compatibility will be preserved.

Reply:
Do we need to warn users that setting this while having tables in the public schema will break in-progress transactions?

Reply:
We can. But this is like setting the different database may result in a different set of tables. I feel it is part of common sense. I am feeling very strong of adding or not adding the comments. WDYT?

Reply:
It's better to be explicit in my opinion.